### DISCUSSION TITLE
Are "must" queries supported?

### DISCUSSION THREAD

**Discussion by wsw70** [zinclabs/zinc#228] [Q&A] [answered]
I need to make a query for documents whose field `type` is equal to the string `journal`. I tried

```
POST https://zinc.example.com/api/myindex/_search HTTP/1.1

}
    "max_results": 1000,
    "query": {
      "bool": {
        "must": [
          {"term": {"type": "journal"}}
        ]
      }
    }
}
```

No matter what I put in the query, I get all the documents. Is `bool` and `must` supported?

**Comment by hengfeiyang** [comment, marked as answer]
Please use ES compatible API:

https://docs.zincsearch.com/api-es-compatible/search/search/

**Reply by wsw70** [reply]
Thank you!
